modular_skyrat/modules/morgue_radio/_define_my_soul_and_give_me_str.dm

```diff
@@ -0,0 +1,2 @@
+#define COMSIG_MORGUE_ALARM "morgue_alarm"
```

Comment:
ew

move this to where other skyrat siganls are

Reply:
where is that

Reply:
found it